howl/settings.py

```diff
@@ -84,4 +116,14 @@ def training(self) -> TrainingSettings:
         return self._training
 
 
-SETTINGS = LazySettingsSingleton()
+KEY_TO_SETTINGS_CLASS = {
+    '_audio': AudioSettings,
+    '_audio_transform': AudioTransformSettings,
+    '_inference_engine': InferenceEngineSettings,
+    '_raw_dataset': RawDatasetSettings,
+    '_dataset': DatasetSettings,
+    '_cache': CacheSettings,
+    '_training': TrainingSettings,
+}
```

Comment:
any particular reason it starts with _?

Reply:
Oh yeah so the original class stores the actual setting objects with _ since the non-underscored property is a function that lazy-loads settings after they are accessed. I just kept the lazy-loading here